### PR TITLE
Beginner level

### PR DESCRIPTION
**Pull request by Mhlengeni**
I have added a beginner level page and linked it to the game levels page. Please review it and merge it to the main branch.

### CODE REVIEW

File: SRC/Views/beginnerLevel.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <title>Beginner Level</title>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=0,user-scalable=no"
+    />
+    <link rel="stylesheet" href="/SRC/Public/Styles/beginnerLevel.css" />
+    <link rel="stylesheet" href="/SRC/Public/Styles/keyboard.css">
+
```
Comment (stallon22) [inline SRC/Views/beginnerLevel.html:13]: Everything about your code is fine, but you should implement routing for the new files that you have added

File: SRC/Views/beginnerLevel.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```
Comment (stallon22) [inline SRC/Views/beginnerLevel.html:1]: Also create  a global variable called "user_login" in mainRoutes and set it to true, like "let user_login = true", you can create it at the top of the file, this is so we can check that your code works.

File: SRC/Views/beginnerLevel.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+<html lang="en">
+  <head>
+    <title>Beginner Level</title>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
```
Comment (stallon22) [inline SRC/Views/beginnerLevel.html:7]: Also, check that your code works at http://127.0.0.1:3000/, before you can make a pull request

**Review by stallon22 — CHANGES_REQUESTED** [review]

**Comment by Mhlengeni** [comment]
I will make changes, commit, and then create another pull request.

**Pull request closed without merging**